OpenMcdf.Ole/PropertySet.cs

```diff
@@ -13,13 +13,27 @@ internal sealed class PropertySet
     public void LoadContext(int propertySetOffset, BinaryReader br)
     {
         long currPos = br.BaseStream.Position;
+
+        // Read the code page - this should always be present
         int codePageOffset = (int)(propertySetOffset + PropertyIdentifierAndOffsets.First(pio => pio.PropertyIdentifier == 1).Offset);
         br.BaseStream.Seek(codePageOffset, SeekOrigin.Begin);
 
         var vType = (VTPropertyType)br.ReadUInt16();
         br.ReadUInt16(); // Ushort Padding
         PropertyContext.CodePage = (ushort)br.ReadInt16();
 
+        // Read the Locale, if present
+        PropertyIdentifierAndOffset? localeProperty = PropertyIdentifierAndOffsets.FirstOrDefault(pio => pio.PropertyIdentifier == 0x80000000);
```

Comment:
It's not necessarily related to this commit, since it happens elsewhere, but it would be nice to have a constant for LOCALE_PROPERTY_IDENTIFIER (renamed to follow dotnet conventions of course) rather than using a magic number...

Reply:
There's a similar situation with the Behavior proprerty as well, though nothing actually checks that as it stands